index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
Change variable names to correspond with names of their colors, like "p" for purple.